routes/auth.js

```diff
@@ -67,4 +68,22 @@ authRouter.get('/logout', (req, res, next) => {
     });
 });
 
+authRouter.get('/check-user', (req, res, next) => {
```

Comment:
consider using `req.params` or smth like that. we already have a middleware for parsing params.

Reply:
fixed